src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/PropertyGridTests.cs

```diff
@@ -0,0 +1,493 @@
+        _propertyGrid.CommandsForeColor.Should().Be(Color.Red);
+    }
+
+    [WinFormsFact]
+    public void PropertyGrid_ContextMenuDefaultLocation_GetSet_ReturnsExpected()
+    {
+        _propertyGrid.ContextMenuDefaultLocation.Should().NotBeNull();
+
+        Point oldLocation = _propertyGrid.ContextMenuDefaultLocation;
+        Point newLocation = new(10, 20);
+
+        _propertyGrid.Location = newLocation;
+        _propertyGrid.ContextMenuDefaultLocation.X.Should().Be(oldLocation.X + newLocation.X);
+        _propertyGrid.ContextMenuDefaultLocation.Y.Should().Be(oldLocation.Y + newLocation.Y);
+    }
+
+    [WinFormsFact]
+    public void PropertyGrid_Controls_GetSet_ReturnsExpected()
+    {
+        _propertyGrid.Controls.Should().NotBeNull();
+
+        using TextBox textBox = new();
+        _propertyGrid.Controls.Add(textBox);
+        _propertyGrid.Controls.Contains(textBox).Should().BeTrue();
+    }
+
+    [WinFormsFact]
+    public void PropertyGrid_HelpBackColor_GetSet_ReturnsExpected()
+    {
+        _propertyGrid.HelpBackColor.Should().Be(SystemColors.Control);
+
+        _propertyGrid.HelpBackColor = Color.Blue;
+        _propertyGrid.HelpBackColor.Should().Be(Color.Blue);
+    }
+
+    [WinFormsFact]
+    public void PropertyGrid_HelpForeColor_GetSet_ReturnsExpected()
+    {
+        _propertyGrid.HelpForeColor.Should().Be(SystemColors.ControlText);
+
+        _propertyGrid.HelpForeColor = Color.Green;
+        _propertyGrid.HelpForeColor.Should().Be(Color.Green);
+    }
+
+    [WinFormsFact]
+    public void PropertyGrid_LineColor_GetSet_ReturnsExpected()
+    {
+        _propertyGrid.LineColor.Should().Be(SystemColors.InactiveBorder);
+
+        _propertyGrid.LineColor = Color.Red;
+        _propertyGrid.LineColor.Should().Be(Color.Red);
+    }
+
+    [WinFormsFact]
+    public void PropertyGrid_PropertySort_GetSet_ReturnsExpected()
+    {
+        _propertyGrid.PropertySort.Should().Be(PropertySort.CategorizedAlphabetical);
+
+        _propertyGrid.PropertySort = PropertySort.Alphabetical;
+        _propertyGrid.PropertySort.Should().Be(PropertySort.Alphabetical);
+    }
+
+    [WinFormsFact]
+    public void PropertyGrid_SelectedObject_GetSet_ReturnsExpected()
+    {
+        _propertyGrid.SelectedObject.Should().Be(_form);
+
+        using TextBox textBox1 = new();
+        _propertyGrid.SelectedObject = textBox1;
+        _propertyGrid.SelectedObject.Should().Be(textBox1);
+
+        _propertyGrid.SelectedObject = null;
+        _propertyGrid.SelectedObject.Should().BeNull();
+    }
+
+    [WinFormsFact]
+    public void PropertyGrid_SelectedObjects_GetSet_ReturnsExpected()
+    {
+        _propertyGrid.SelectedObjects.Should().Contain(_form);
+
+        object[] objects = [new Button(), new TextBox(), new ComboBox()];
+        _propertyGrid.SelectedObjects = objects;
+        _propertyGrid.SelectedObjects.Should().Contain(objects);
+
+        _propertyGrid.SelectedObjects = Array.Empty<object>();
```

Comment:
consider simplifying it.
```suggestion
        _propertyGrid.SelectedObjects = [];
```